code/game/objects/structures/spirit_board.dm

```diff
@@ -6,77 +6,99 @@
 	resistance_flags = FLAMMABLE
 	density = TRUE
 	anchored = FALSE
+	/// Whether no one has moved the planchette yet.
 	var/virgin = TRUE //applies especially to admins
-	var/next_use = 0
-	var/planchette = "A"
+	/// How long between planchette movements.
+	COOLDOWN_DECLARE(next_use)
+	/// Where the planchette is currently pointing.
+	var/planchette
+	/// Ckey of last mob to use the board.
 	var/lastuser = null
+	/// List of options ghosts (or people) can pick from.
+	var/list/ghosty_options = list(
+		"A","B","C","D","E","F","G","H","I","J","K","L","M","N","O","P","Q","R","S","T","U","V","W","X","Y","Z",
+		"1","2","3","4","5","6","7","8","9","0",
+		"Yes","No",
+	)
+	/// Admin only var, allows admins to bypass the check requiring multiple players
+	var/allow_oneman_use = FALSE
+
+/obj/structure/spirit_board/Initialize(mapload)
+	. = ..()
+	if(prob(1))
+		name = "luigi board"
+	planchette = ghosty_options[1]
 
 /obj/structure/spirit_board/examine()
-	desc = "[initial(desc)] The planchette is sitting at \"[planchette]\"."
 	. = ..()
+	. += span_notice("The planchette is currently at the letter \"[planchette]\".")
```

Comment:
we should have a case for when `planchette` is null because otherwise this message will look weird